package.json

```diff
@@ -64,9 +64,6 @@
       "verbosity": 1
     }
   },
-  "optionalDependencies": {
```

Comment:
This is sadly still needed. There are optional dependencies of some development dependencies that when added to the shrinkwrap file, will cause the other platforms to complain about extraneous dependencies.